app/controllers/api/github_issues_controller.rb

```diff
@@ -3,11 +3,13 @@ class Api::GithubIssuesController < ApplicationController
   before_action :create_github_client, only: [:index, :create]
 
   def index
-    @issues = @client.issues.list
+    @issues = @client.issues.list user: 'TeamMartinez', repo: 'YOLO-api', state: 'open'
     render json: @issues
   end
 
   def create
+    # TODO: make user: and repo: params default somewhere and use params
+    @client.issues.create user: 'TeamMartinez', repo: 'YOLO-api', title: params[:title]
     render json: { errors: "not implemented" }
```

Comment:
I want to use `issues_params` to make `title` required and pass it in to `issues.create`